scripts/exploration.gd

```diff
@@ -51,7 +51,7 @@ func enemyencounter():
 	if patrol != 'none':
 		text = encounterdictionary(enemygroup.description) + "Your bad reputation around here will certainly lead to a difficult fight..."
 		encounterbuttons(patrol)
-	elif scoutawareness < enemyawareness:
+	elif scoutawareness < (enemyawareness + rand_range(-7,7)): #ralph4
```

Comment:
changed balance of awareness without condition

Reply:
Fixed